flecs_ecs/Cargo.toml

```diff
@@ -50,6 +50,14 @@ flecs_meta = []
 flecs_metrics = []
 flecs_ecs_asserts = []
 flecs_enable_all_asserts = ["flecs_ecs_asserts"]
+
+# Enabling this will not build a copy of flecs into this library.
+# Instead, the executable that this is linked with will need to
+# provide the symbols required. This is useful when using both
+# Rust and C++ code in an application and a copy of flecs is
+# already being linked into the executable via other means.
+flecs_use_runtime_flecs = []
```

Comment:
I'm not a fan of the naming of the feature flag. 

flecs_use_runtime_flecs makes me think about if it'll use flecs under the hood, which it does either way wherever it's build. 

What do you think of `flecs_disable_build_c_library` or similar?

Reply:
Okay, will rename it.